### PR TITLE
Update UG

### LINKED ISSUE #150: Update UG for CS2101
Just follow guide set out in cs2101

### PR DESCRIPTION
**Pull request by li-s**
Update UG for 2101
Resolve #150

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -1,23 +1,31 @@
...
-1. Download the latest hellofile.jar.
-1. Copy the file to the folder you want to use as the home folder for HelloFile to start.
-1. Double-click the file to start the app.
-1. Type the command in the command box and press Enter to execute it. e.g. typing help and pressing Enter will open the help window.
+### Installation
+1. Ensure you have [Java 11](https://www.java.com/en/download/) or above installed in your computer. 
+2. Download the latest HelloFile.jar [here](https://github.com/AY2021S1-CS2103T-F12-1/tp/releases).
 
+### Quick start
+1. Move the file to the folder you intend to use as the home folder for HelloFile.
```
Comment (luo-git) [inline docs/UserGuide.md]: The file can be changed to HelloFile.jar?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -28,36 +36,43 @@ HelloFile is a desktop app for managing files, optimised for use via a Command L
...
+* Every tag name is case-sensitive. e.g tag name `notes` is different from tag name `Notes`.
+
+* Only the `cd` command accepts relative file path, all other commands require absolute file path.
 
 </div>
 
 ### Viewing help : `help`
 
-Shows a command's usage and format, if no command is specified, show help for all commands.
+Display help for all commands.
```
Comment (luo-git) [inline docs/UserGuide.md]: Displays

File: docs/UserGuide.md [hunk 3]
```diff
@@ -77,7 +92,7 @@ Examples:
 
 ### Removing a tag : `untag`
 
-Removes the file from the list of managed files.
+Removes the tag from the list of managed tag names.
```
Comment (luo-git) [inline docs/UserGuide.md]: managed tag names -> managed tags?

File: docs/UserGuide.md [hunk 4]
```diff
@@ -86,24 +101,31 @@ Examples:
 
 ### Renaming a tag : `retag`
 
-Renames the tag of the file with a new tag.
+Renames the unique tag name of the file.
```
Comment (luo-git) [inline docs/UserGuide.md]: "Renames a tag's name"?

File: docs/UserGuide.md [hunk 5]
```diff
@@ -116,24 +138,29 @@ Format: `exit`
...
 ## FAQ
-**Q**: How do I transfer my data to another Computer?<br>
-**A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous HelloFile home folder.
-
 **Q**: What if the name or the directory of the file I tagged is changed? Can I still access the file using HelloFile?<br>
-**A**: No. HelloFile cannot trace the file if its name or directory is changed, but if you still want to manage the file, you can tag it again.
+**A**: No. HelloFile is currently unable to track a file if its name or directory is changed. It is necessary to delete the old tag and retag the file.
+
+**Q**: What is the structure of the application?<br>
+**A**: It can be found in the Degisn Guide [here](https://github.com/AY2021S1-CS2103T-F12-1/tp/blob/master/docs/DeveloperGuide.md)
```
Comment (luo-git) [inline docs/UserGuide.md]: Typo in design?

**Review by luo-git — CHANGES_REQUESTED** [review]
Looks good. Some changes required.

Generally, use "you can", instead of "it is".
E.g. "It is necessary" -> "You will need to".

File: docs/UserGuide.md (hunk 5 above)
Comment (li-s) [reply docs/UserGuide.md]: Ok

File: docs/UserGuide.md (hunk 3 above)
Comment (li-s) [reply docs/UserGuide.md]: Ok

**Review by luo-git — APPROVED** [review]
LGTM

**Pull request merged**